src/cdk/testing/protractor/protractor-element.ts

```diff
@@ -206,8 +206,8 @@ export class ProtractorElement implements TestElement {
   }
 
   /** Gets the value of a property of an element. */
-  async getProperty(name: string): Promise<any> {
-    return browser.executeScript(`return arguments[0][arguments[1]]`, this.element, name);
+  async getProperty<T = any>(name: string): Promise<T> {
```

Comment:
ideally we'd make the default `unknown` but I guess maybe that's too breaking?

Reply:
Yeah, I suspect that `unknown` will be breaking. I doubt that `getProperty` is used too often since most of the time you can get the same information through `getAttribute`.